.swiftformat

```diff
@@ -69,7 +69,8 @@
     spaceInsideComments, \
     spaceInsideGenerics, \
     spaceInsideParens, \
-    strongOutlets
+    strongOutlets, \
```

Comment:
🤔

Reply:
There was a bit of conversation about this. Apparently this is what apple advises in most cases now. However in practice we don't have any weak IBOutlets in our codebase and probably won't get any.